### DISCUSSION TITLE
add an option for CR3 file generation

### DISCUSSION THREAD

**Discussion by vvpeter** [dnglab/dnglab#446] [Ideas]
For Canon burst-mode files, add an option to generate the CR3 RAW files of each frame rather than just DNGs (or in addition to).

**Comment by cytrinox** [comment]
This is not planned.

**Reply by vvpeter** [reply]
That's fine - Canon finally added an option to their app to save out all the burst images in separate raw files in a single step.

**Reply by cytrinox** [reply]
In Canon DPP? Good to know.
